From 9f53bde61177f49c468cd41600ef11cb5438e769 Mon Sep 17 00:00:00 2001
From: Zheng Junjie <zhengjunjie@iscas.ac.cn>
Date: Wed, 9 Oct 2024 13:32:49 +0800
Subject: [PATCH 2/2] Revert "add cstdint header"

This reverts commit 012bc63774e495bc2295e1a000312d3fbd6de378.

upstream fix it.
---
 debian/patches/01-add-cstdint.patch | 10 ----------
 debian/patches/series               |  1 -
 2 files changed, 11 deletions(-)
 delete mode 100644 debian/patches/01-add-cstdint.patch
 delete mode 100644 debian/patches/series

diff --git a/debian/patches/01-add-cstdint.patch b/debian/patches/01-add-cstdint.patch
deleted file mode 100644
index 80a9f18..0000000
--- a/debian/patches/01-add-cstdint.patch
+++ /dev/null
@@ -1,10 +0,0 @@
---- a/include/andino_base/motor_driver.h
-+++ b/include/andino_base/motor_driver.h
-@@ -30,6 +30,7 @@
- #pragma once
- 
- #include <array>
-+#include <cstdint>
- 
- #include <libserial/SerialPort.h>
- 
diff --git a/debian/patches/series b/debian/patches/series
deleted file mode 100644
index 73c9664..0000000
--- a/debian/patches/series
+++ /dev/null
@@ -1 +0,0 @@
-01-add-cstdint.patch
-- 
2.46.0

